### PR TITLE
fix submit and checkbox of terms and conditions

### PR DESCRIPTION
**Pull request by samaradel**
- https://github.com/codescalers/cloud4students/issues/230
- https://github.com/codescalers/cloud4students/issues/231

### CODE REVIEW

File: client/src/views/Signup.vue [hunk 1]
```diff
@@ -181,6 +181,7 @@ export default {
       (value) => {
         if (!value) return "Field is required";
         if (value < 1) return "Team Size should at least 1";
```
Comment (rawdaGastan) [inline client/src/views/Signup.vue]: `Team Size should be at least 1` add `be`

**Review by rawdaGastan — CHANGES_REQUESTED** [review]
I don't think https://github.com/codescalers/cloud4students/issues/230 is resolved?

**Comment by samaradel** [comment]
> I don't think #230 is resolved?

[Screencast from 14-05-23 16:09:35.webm](https://github.com/codescalers/cloud4students/assets/11662805/6e2046db-5c89-4cb0-bebb-ac1290c940ab)

**Comment by samaradel** [comment]
> I don't think #230 is resolved?

as per our discussion, it's fixed now

File: client/src/views/Signup.vue [hunk 2]
```diff
@@ -161,10 +161,9 @@ export default {
     const loading = ref(false);
     const toast = ref(null);
     const checked = ref(false);
-    const nameRegex = /^(\w+\s){0,3}\w*$/;
     const nameValidation = ref([
       (value) => {
-        if (!value.match(nameRegex)) return "Must be at most four names";
+        if (!value) return "Field is required";
```
Comment (rawdaGastan) [inline client/src/views/Signup.vue:166]: why did you remove it?
`if (!value.match(nameRegex)) return "Must be at most four names";`

**Review by rawdaGastan — APPROVED** [review]

File: client/src/views/Signup.vue (hunk 2 above)
Comment (samaradel) [reply client/src/views/Signup.vue:166]: removed by mistake, fixed now

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**